frontend/containers/open-call-page/header/component.tsx

```diff
@@ -151,10 +151,16 @@ export const OpenCallHeader: FC<OpenCallHeaderProps> = ({ openCall, instrumentTy
                 <hr className="mt-6 mb-8" />
                 <div className="flex flex-col justify-between gap-8 pl-2 sm:flex-row sm:gap-11">
```

Comment:
When the circle is shown, the alignment and spacing is not exactly like the design. Can you have a look?
<table>
<thead>
	<td><b>Designs</b>
	<td><b>Reality</b>
<tr>
	<td>
<img src="https://user-images.githubusercontent.com/6073968/195633403-dabaa5df-061f-449f-9e42-2362e0c622a3.png" alt="The deadline and circle are aligned in the middle" width="300" />
	<td>
<img src="https://user-images.githubusercontent.com/6073968/195633145-2904334a-45ed-4c83-94b3-aac036c6863b.png" alt="The deadline and circle are aligned at the bottom" width="300" />

</table>